Bench.php

```diff
@@ -311,6 +315,43 @@ public function getLongest()
     }
 
     /**
+     * Pause all Marks.
+     *
+     * @return  array
+     */
+    public function pause()
+    {
+        $runningMarks = [];
+
+        foreach ($this as $mark) {
+            if ($mark->isRunning()) {
+                $runningMarks[] = $mark;
+            }
+        }
+
+        // short loop = reduce delta between pauses
+        // same loop as in resume()
+        foreach ($runningMarks as $mark) {
+            $mark->pause(true);
```

Comment:
I know what `true` means here, I am just asking why forcing it :-p? It it is the default parameter, then, no need to put it. We are not going to change it.